pkg/provider/cloud/openstack/helper.go

```diff
@@ -203,6 +204,15 @@ func createKubermaticSecurityGroup(netClient *gophercloud.ServiceClient, cluster
 			PortRangeMax: provider.DefaultSSHPort,
 			Protocol:     osecuritygrouprules.ProtocolTCP,
 		},
+		{
+			// Allows NodePort range
+			Direction:    osecuritygrouprules.DirIngress,
+			EtherType:    osecuritygrouprules.EtherType4,
+			SecGroupID:   securityGroupID,
+			PortRangeMin: 30000,
+			PortRangeMax: 32000,
```

Comment:
Isn't the NodePortRange configurable in the [Cluster Controller](https://github.com/kubermatic/kubermatic/blob/6b0a6c05b2592de605e6f0871cdee16e565e57d3/pkg/controller/seed-controller-manager/kubernetes/cluster_controller.go#L82)? In that case, we need to put that value here.

Additionally, there's a PR (#6533) to make the NodePortRange configurable using ComponentOverrides.